### PR TITLE
setup React 

### PR DESCRIPTION
**Pull request by andym80**
"This pull request adds a basic React frontend "Hello World" feature that is nested within an existing Rails app backend. The front end is built using React and is connected to the backend using an API. 
The feature can be accessed by navigating to the '/Greeting' route in the Rails app. 
This feature serves as a starting point for future development of the application's front end.
 Please review and test thoroughly before merging."

### CODE REVIEW

**Comment by addod19** [comment]
### Hi @andym80, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that did not meet the minimum requirements. Here are the reasons; 
- [ ] Linters not set up for our frontend project
- [ ] README file is not descriptive and missing the main headers as we have in our backend

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Review by Nemwel-Boniface — CHANGES_REQUESTED** [review]
## Hello @andym80         :wave: , 
I'm Nemwel your reviewer for this PR :man_beard: 

Good job working on your project so far! :clap: 
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights

Great job setting up the correct linters ✔️ 
Professional documentation ✔️ 
Correct workflow ✔️ 
Meaningful commit messages ✔️  


## ♻️ Required Changes ♻️

**As I am not able to highlight the piece of code that needs changing, I will include it with screenshots to supplement it.**

- [ ] The line in your Redux reducer.js line 25 seems to be wrong. According to your API, you need the ```api/greetings``` and not ```/greetings``` to be able to get data from your endpoint. Kindly fix this.
- [ ] Also, in your App.js file, you are not supposed to import Home as the component is called Greeting that you intend to use. Kindly fix this.


_Check the comments under the review._

## [Optional] suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding! :partying_face: 👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me **@Nemwel-Boniface** in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/redux/reducer.js [hunk 1]
```diff
@@ -16,13 +16,13 @@ const getGreetingsSucces = () => ({
...
-  payload: "Error",
+  payload: 'Error',
 });
 
 // Thunk
 
 const getGreeting = () => async (dispatch) => {
-  const response = await axios.get("http://localhost:3001/greetings");
+  const response = await axios.get('http://localhost:3001/api/greetings');
   const greeting = response.data;
```
Comment (Ismailco) [inline src/redux/reducer.js:26]: - [ ] [OPTIONAL] Greate work creating the app however you should get only the message from the API so your app is working 
```suggestion
  const greeting = response.data.message;
```

**Review by Ismailco — APPROVED** [review]
### Hi @andym80, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

cr-approve


## Highlights

✔️  No linter errors
✔️ Used Gitflow
✔️ Work documented professionally

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**